src/interfaces/components/loginPage.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface LoginPageProps {
+	descopeRenderKey: number;
+	handleSuccess: (event: CustomEvent<any>) => Promise<void>;
+	t: (key: string, options?: any) => string;
```

Comment:
Why did you add `t` in prop? i18n can be called inside a component

Reply:
removed